server.go

```diff
@@ -44,8 +44,8 @@ import (
 const (
 	// defaultServices describes the default services that are supported by
 	// the server.
-	defaultServices = wire.SFNodeNetwork | wire.SFNodeBloom |
-		wire.SFNodeWitness | wire.SFNodeCF
+	defaultServices = wire.SFNodeNetwork | wire.SFNodeNetworkLimited |
+		wire.SFNodeBloom | wire.SFNodeWitness | wire.SFNodeCF
```

Comment:
If pruning isn't active, then this bit should be removed in `newServer`: 
https://github.com/btcsuite/btcd/blob/85687d455d8cca1077dae5fbf20accd59cb00da9/server.go#L2726-L2733

Reply:
In Core, `NODE_NETWORK_LIMITED` is always on and `NODE_NETWORK` bit is added if the node isn't pruned.

https://github.com/bitcoin/bitcoin/blob/a4ca4975880c4f870c6047065c70610af2529e74/src/init.cpp#L786

https://github.com/bitcoin/bitcoin/blob/a4ca4975880c4f870c6047065c70610af2529e74/src/init.cpp#L1598C13-L1601

This is reflected in bitnodes.io for archival nodes https://bitnodes.io/nodes/18.27.79.17-8333/

I think it's better to follow the behavior of Core nodes since it's what everyone expects.